package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "audit-service",
-  "version": "0.1.3",
+  "version": "0.1.7",
```

Comment:
today the tag version is 0.1.6 and it always should be the same as latest, after i will release a new tag then it will be 0.1.7